src/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard card</title>
+</head>
+
+<body>
+
+    <!-- <div class="github">
```

Comment:
- [ ] Kindly remove all the comments code from the project, it makes the project look unclean.